<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>881</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>80</y>
      <width>261</width>
      <height>179</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="usernameLabel">
       <property name="text">
        <string>Kullanıcı Adı</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="passwordLabel">
       <property name="text">
        <string>Parola</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="loginButton">
       <property name="text">
        <string>Giriş</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QPushButton" name="logoutButton">
       <property name="text">
        <string>Güvenli Çıkış</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="usernameLineEdit">
       <property name="text">
        <string>33333303</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="passwordLineEdit"/>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>40</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>GİB Kullanıcı İşlemleri</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>280</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Dosya Seç</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>360</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Toplam Satır Sayısı</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>370</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Faturalama Tipini Seçiniz</string>
    </property>
   </widget>
   <widget class="QPushButton" name="createInvoiceButton">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>380</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Faturaları Oluştur</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="mainDataTable">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>440</y>
      <width>831</width>
      <height>192</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>TCKN/VKN</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>İSİM</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>KDV %</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>KDV</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>KDV HARİÇ</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>TOPLAM</string>
     </property>
    </column>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>400</y>
      <width>171</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Her Satır Ayrı Fatura</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Bla Bla</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>410</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Faturaların Detayları</string>
    </property>
   </widget>
   <widget class="QPushButton" name="displayInvoicesButton">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>100</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Fatura Görüntüle</string>
    </property>
   </widget>
   <widget class="QPushButton" name="filePathButton">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>280</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Dosya Bul</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="filePathBrowser">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>280</y>
      <width>501</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>330</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Seçilen Dosya</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>330</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>881</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
